dict/french.yml

```diff
@@ -0,0 +1,291 @@
+
+  SOUTH:
+    'AF': [Afghanistan, Afghan*, Kaboul]
+    'BD': [Bangladesh, Bengali*, Bangladais*, Dhaka, Dacca]
+    'BT': [Bhoutan, Bhoutanais*, Thimphou, Thimphu]
+    'IN': [Inde, Indien*, Mumbai, Bombay, New Delhi]
+    'IR': [Iran, Iranien*, Téhéran]
+    'LK': [Sri Lanka, Sri Lankais*, Colombo]
+    'MV': [Maldives, Maldivien*]
+    'NP': [Népal, Nepalais*, Katmandu, Kathmandu, Katmandou]
+    'PK': [Pakistan, Pakistanais*, Islamabad]
+
+  SOUTH-EAST:
+    'BN': [Brunei, Brunéien*]
+    'ID': [Indonesie, Indonesien*, Jakarta, Djakarta]
+    'KH': [Cambodge, Cambodgien*, Phnom Penh]
+    'LA': [Laos, Laotien*, Vientiane]
+    'MM': [Myanmar, Birmanie, Birman*, Rangoon, Yangon, Naypyidaw]
+    'MY': [Malaisie, Malaisien*, Kuala Lumpur, Putrajaya]
+    'PH': [Philippines, Philippin*, Manille]
+    'SG': [Singapour, Singapourien*]
+    'TH': [Thaïlande, Thaïlandais*, Bangkok]
+    'TL': [Timor oriental, Timor Leste, Timorais, Dili]
+    'VN': [Viêt Nam, Viet Nam, Vietnam, Viêtnam, Vietnamien*, Hanoi, Hanoï, Ho Chi Minh Ville, Hô-Chi-Minh-Ville]
+
+  WEST:
+    'AE': [Émirats arabes unis, Emirats arabes unis, ÉAU, Émirati*, Émirien*, Dubai, Dubaï, Abu Dhabi, Abou Dabi]
+    'AM': [Arménie, Arménien*, Erevan]
+    'AZ': [Azerbaïdjan, Azéri*, Azerbaïdjanais*, Bakou]
+    'BH': [Bahrain, Bahreïn, Bahreïnien*, Manama]
+    'CY': [Chypre, Chypriot*, Nicosie]
+    'GE': [Georgie, Georgien*, Tbilisi, Tbilissi]
+    'IL': [Israel, Israël, Israélien*, Jérusalem]
+    'IQ': [Irak, Irakien*, Bagdad]
+    'JO': [Jordanie, Jordanien*, Amman]
+    'KW': [Koweït, Koweïtien, Koweit, Koweitien*, ville de Koweit]
+    'LB': [Liban, Libanais*, Beyrouth]
+    'OM': [Oman, Omanais*, Mascate]
+    'PS': [Palestine, Palestinien*, Gaza, Gazaoui*, Cisjordanie, Cisjordanien*]
+    'QA': [Qatar, Qatarien*, Qatari*, Qatariote*, Doha]
+    'SA': [Arabie Saoudite, Saoudien*, Riyadh, Riyad]
+    'SY': [Syrie, Syrien*, Damas]
+    'TR': [Turquie, Turc*, Turque*, Ankara, Istanbul]
+    'YE': [Yemen, Yémen, Yemeni*, Yémeni*, Sanaa]
+
+EUROPE:
+  EAST:
+    'BG': [Bulgarie, Bulgare*, Sofia]
+    'BY': [Belarus, Biélorussie, Biélorusse*, Minsk]
+    'CZ': [République Tchèque, république tchèque, Tchéquie, Tchèque*, Prague]
+    'HU': [Hongrie, Hongrois*, Budapest]
+    'MD': [Moldavie, Moldave*, Chisinau]
+    'PL': [Pologne, Polonais*, Varsovie]
+    'RO': [Roumanie, Roumain*, Bucarest]
+    'RU': [Russie, Russe*, Moscou]
+    'SK': [Slovaquie, Slovaqu*, Bratislava]
+    'UA': [Ukraine, Ukrainien*, Kiev]
+
+  NORTH:
+    'AX': [Aaland, Åland, Ålandais*, Aalandais, Mariehamn]
+    'DK': [Danemark, Danois*, Copenhague]
+    'EE': [Estonie, Estonien*, Tallinn]
+    'FI': [Finlande, Finlandais, Finnois*, Helsinki]
+    'FO': [Féroé , Féroïen*, Torshavn]
+    'GB': [Royaume-Uni, Grande-Bretagne, Anglais*, Britannique*, Londres]
+    'GG': [Guernesey, Guernesiais*, Saint Peter Port, St Peter Port]
+    'IE': [Irelande, Irlandais*, Dublin]
```

Comment:
@ClaudeGrasland I suspect that there is a typo in Ireland. Can you double check?